apiserver/listener_test.go

```diff
@@ -157,14 +158,17 @@ func (s *listenerSuite) TestPause(c *gc.C) {
 	}()
 
 	start <- true
-	s.clock.Advance((l.minPause/time.Millisecond - 1) * time.Millisecond)
+	minPauseTimeGone := time.After(50 * time.Millisecond)
+	err := s.clock.WaitAdvance(time.Millisecond, (l.minPause/time.Millisecond-1)*time.Millisecond, 1)
```

Comment:
use coretesting.ShortWait for the 2nd arg here? i.e. the amount of time to when something is expected *not* to happen